operator/pkg/helmreconciler/prune.go

```diff
@@ -61,11 +70,34 @@ var (
 		// Cannot currently prune CRDs because this will also wipe out user config.
 		// {Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: name.CRDStr},
 	}
+
+	nonNamespacedCPResources = []schema.GroupVersionKind{
+		{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: name.MutatingWebhookConfigurationStr},
+	}
+	namespacedCPResources = []schema.GroupVersionKind{
```

Comment:
Missing envoyfilters. Maybe it should be a list of resources we *don't* want to prune rather than ones we do

Reply:
I think that would work for the case @esnible mentioned below if we have the support for --filename or config-from-cluster flag during uninstall, for that we can generate the manifests and prune everything other than the exclusion part. But if user just provide a revision flag, we can't do an exclusion easily because we don't have the generated manifest, right? WDYT? @howardjohn @ostromart